common/src/main/resources/assets/voicechat/lang/pl_pl.json

```diff
@@ -1,103 +1,165 @@
-  "message.voicechat.recording_started": "Rozpoczęto nagrywanie",
-  "message.voicechat.recording_stopped": "Zakończono nagrywanie",
-  "message.voicechat.open_folder": "Otwórz folder",
-  "message.voicechat.storage_size": "Spodziewana przestrzeń na dysku: %s",
-  "message.voicechat.saved_debug_report": "Zapisano raport debugowania czatu głosowego do \"%s\"",
-  "message.voicechat.open": "Otwórz",
-  "message.voicechat.saved_debug_report_failed": "Nie udało się zapisać raportu debugowania czatu głosowego: %s",
-  "message.voicechat.microphone_unavailable": "Mikrofon niedostępny",
-  "message.voicechat.playback_unavailable": "Ponowne odtwarzanie niedostępne",
-  "message.voicechat.denoiser": "Tłumienie hałasu: %s",
-  "message.voicechat.enabled": "Włączono",
-  "message.voicechat.disabled": "Wyłączono",
-  "message.voicechat.group_does_not_exist": "Ta grupa nie istnieje",
-  "message.voicechat.create": "Stwórz",
-  "message.voicechat.create_group": "Stwórz grupę",
-  "message.voicechat.create_group_button": "Stwórz grupę",
-  "message.voicechat.optional_password": "Hasło (nieobowiązkowe)",
-  "message.voicechat.join_group": "Dołącz do grupy",
-  "message.voicechat.enter_group_password": "Wpisz hasło grupy",
-  "message.voicechat.password": "Hasło",
-  "message.voicechat.wrong_password": "Niepoprawne hasło",
-  "message.voicechat.server_port": "Serwer czatu głosowego hostowany na porcie %s",
-  "message.voicechat.macos_no_mic_permission": "Brak uprawnień do korzystania z mikrofonu. Proszę sprawdzić ustawienia prywatności.",
-  "cloth_config.voicechat.settings": "Ustawienia czatu głosowego",
-  "cloth_config.voicechat.category.other": "Inne",
-  "cloth_config.voicechat.category.audio": "Dźwięk",
-  "cloth_config.voicechat.category.hud_icons": "Ikony HUD",
-  "cloth_config.voicechat.category.group_chat_icons": "Ikony czatu grupowego",
-  "cloth_config.voicechat.config.audio_packet_threshold": "Próg pakietów audio",
-  "cloth_config.voicechat.config.deactivation_delay": "Czas dezaktywacji",
-  "cloth_config.voicechat.config.output_buffer_size": "Wielkość buforu wyjściowego",
-  "cloth_config.voicechat.config.recording_destination": "Folder docelowy nagrań",
-  "cloth_config.voicechat.config.run_local_server": "Działaj w trybie jednoosobowym/światach LAN",
-  "cloth_config.voicechat.config.hud_icon_scale": "Wielkość ikon HUD",
-  "cloth_config.voicechat.config.hud_icon_x": "Pozycja X ikon HUD",
-  "cloth_config.voicechat.config.hud_icon_y": "Pozycja Y ikon HUD",
-  "cloth_config.voicechat.config.group_player_icon_orientation": "Orientacja ikon HUD grupy",
-  "cloth_config.voicechat.config.group_hud_icon_scale": "Wielkość ikon HUD grupy",
-  "cloth_config.voicechat.config.group_player_icon_pos_x": "Pozycja X ikon HUD grupy",
-  "cloth_config.voicechat.config.group_player_icon_pos_y": "Pozycja Y ikon HUD grupy",
-  "cloth_config.voicechat.config.show_own_group_icon": "Wyświetlaj ikony HUD własnej grupy"
-}
```

Comment:
Please properly commit your changes. You replaced every line despite some of them not being changed.